modules/custom/az_carousel/config/install/views.view.az_carousel.yml

```diff
@@ -601,7 +610,6 @@ display:
         fields: false
         sorts: false
         pager_options: false
-        footer: false
```

Comment:
Any idea what this `footer` change is about?  Just want to make sure it's not undoing anything from #3363

Reply:
Yeah, the footer was added here: https://github.com/az-digital/az_quickstart/commit/1b175ea19531325f7585d585556d2ce751867cf7#diff-59a762683ffc2801f3f36e5b61c632fef90614d7ed22155af20edaf893ed7f3bR609

Reply:
https://9d0532dc-9753-4996-8fef-e4697e9c6c4d.probo.build/user/login?destination=/admin/az-carousel/reorder-carousel-items